.config/brix/rust/template.yml

```diff
@@ -0,0 +1,8 @@
+commands:
+  - template:
+      source: template.rs.txt
+      destination: output/template/{{module}}.rs
```

Comment:
I wonder if we could call module `config` with a help description of `config base name`